src/BackgroundTasks/BucketMeta.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace ILIAS\BackgroundTasks;
+
+use ILIAS\BackgroundTasks\Exceptions\Exception;
+use ILIAS\BackgroundTasks\Task\UserInteraction\Option;
+
+/**
+ * Interface Observer
+ *
+ * @package ILIAS\BackgroundTasks
+ *
+ * A meta bucket contains infos about a bucket like its percentage, name etc.
+ */
+interface BucketMeta {
```

Comment:
As far as I understand, this duplicates some methods from the bucket. Let `Bucket` return an instance of this or extend `Bucket` from this. Maybe also rename this to `BucketInfo`.